LICENSE

```diff
@@ -0,0 +1,10 @@
+The OpenTURNS library and all its derivatives are distributed under the LGPL license, see COPYING.
```

Comment:
actually it's COPYING and COPYING.LESSER files ; for the gpl and for the lgpl (lgpl is an extension of the gpl)